m55m1x/Devices/M55M1/Include/M55M1.h

```diff
@@ -1670,7 +1670,8 @@ typedef volatile uint64_t vu64;   ///< Define 64-bit unsigned volatile data type
 #include "qspi.h"
 #include "rng.h"
 #include "rtc.h"
-#include "sc.h"
+/* Not include smart card header file to avoid redefined warning in /bluetooth/host/gatt.c */
+//#include "sc.h"
```

Comment:
how about just deleting the line? the reason for that is in the commit message anyway, no point leaving it in the code

Reply:
Yes, to keep reason in commit message instead of code.
It's done.